src/views/Header.js

```diff
@@ -71,6 +76,24 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     return this._getHeaderTitleString(lastScene);
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    if (!lastScene) {
+      return null;
+    }
+    const {
+      headerBackTitleStyle,
```

Comment:
The `title` is taken from a previous scene as the scene cannot control what scenes are behind it in the stack. However, it's in charge of its own styling and how the status bar appear. 

Existing approach makes it impossible to set back button style of the same scene different on two screens.